releases/latest/beta/helpers/runtime/startupHealthCheck.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+#Default to 1s - Can change through parameter or server env
+timeout_seconds=1
+timeout_milliseconds=$((timeout_seconds * 1000))
+
+#Default location for 'started' file - Can change through parameter or server env
+started_file=/output/health/started
+
+#Static values - will not be changed
+sleep_seconds=0.01
+sleep_milliseconds=10
+
+
+# Evaluate if there is a env var for timeoutSeconds
+if [ -n "${STARTUP_PROBE_TIMEOUT_SECONDS}" ]
+then
+  #Check input is numerical only
+  if [[ ${STARTUP_PROBE_TIMEOUT_SECONDS} =~ ^[0-9]+$ ]]
+  then
+    timeout_seconds=${STARTUP_PROBE_TIMEOUT_SECONDS}
+    timeout_milliseconds=$((timeout_seconds * 1000))
+  else
+    echo "Expected only a numerical value for STARTUP_PROBE_TIMEOUT_SECONDS environment variable, but recieved: $2. This value will not be read in."
```

Comment:
Received value is env variable's value, not `$2`. 
Suggestion: `This value will not be read in.` --> `This value will not be used.`